code/WorkInProgress/recycling/disposal_chute.dm

```diff
@@ -196,47 +196,25 @@
 
 		if(isitem(MO))
 			var/obj/item/I = MO
-
-			if(prob(20)) //It might land!
-				I.set_loc(get_turf(src))
-				if(prob(30)) //It landed cleanly!
-					I.set_loc(src)
-					update()
-					src.visible_message("<span class='alert'>\The [I] lands cleanly in \the [src]!</span>")
-				else	//Aaaa the tension!
-					src.visible_message("<span class='alert'>\The [I] teeters on the edge of \the [src]!</span>")
-					var/delay = rand(5, 15)
-					SPAWN(0)
-						var/in_x = I.pixel_x
-						for(var/d = 0; d < delay; d++)
-							if(I) I.pixel_x = in_x + rand(-1, 1)
-							sleep(0.1 SECONDS)
-						if(I) I.pixel_x = in_x
-					SPAWN(delay)
-						if(I && I.loc == src.loc)
-							if(prob(40)) //It goes in!
-								src.visible_message("<span class='alert'>\The [I] slips into \the [src]!</span>")
-								I.set_loc(src)
-							else
-								src.visible_message("<span class='alert'>\The [I] slips off of the edge of \the [src]!</span>")
-
-		else if (ishuman(MO))
-			var/mob/living/carbon/human/H = MO
-			H.set_loc(get_turf(src))
-			if(prob(30))
-				H.visible_message("<span class='alert'><B>[H] falls into the disposal outlet!</B></span>")
-				logTheThing("combat", H, null, "is thrown into a [src.name] at [log_loc(src)].")
-				H.set_loc(src)
-				if(prob(20))
-					src.visible_message("<span class='alert'><B><I>...accidentally hitting the handle!</I></B></span>")
-					H.show_text("<B><I>...accidentally hitting the handle!</I></B>", "red")
-					flush = 1
-					if (!is_processing)
-						SubscribeToProcess()
-						is_processing = 1
-					update()
-				else
-					update()
+			I.set_loc(src)
+			update()
+			src.visible_message("<span class='alert'>\The [I] lands cleanly in \the [src]!</span>")
+
+		else if (istype(MO, /mob/living))
+			var/mob/living/H = MO
+			H.visible_message("<span class='alert'><B>[H] falls into the disposal outlet!</B></span>")
+			logTheThing("combat", H, null, "is thrown into a [src.name] at [log_loc(src)].")
+			H.set_loc(src)
+			if(prob(10) || H.bioHolder?.HasEffect("clumsy"))
+				src.visible_message("<span class='alert'><B><I>...accidentally hitting the handle!</I></B></span>")
+				H.show_text("<B><I>...accidentally hitting the handle!</I></B>", "red")
+				flush = 1
+				if (!is_processing)
+					SubscribeToProcess()
+					is_processing = 1
+				update()
+			else
+				update()
```

Comment:
```suggestion
			update()
```
I know you were just copying the previous code, but yeah.

Reply:
Shoot yeah I saw that and thought to fix it but then forget !